CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 ## Unreleased
 [Commits](https://github.com/scalableminds/webknossos/compare/19.02.0...HEAD)
 
+### Added
+
+- Added the possibility to create volume annotation tasks. When creating a task type, select skeleton vs volume. Note that compound viewing for volume tasks is not supported yet. Same for creating volume tasks from uploaded nml/data files.
```

Comment:
The vs sounds strange, maybe: "When creating a task type, select whether to create `volume` or `skeleton` tasks.
Also the PR link should be added